src/Microsoft.Health.Dicom.Web/appsettings.json

```diff
@@ -111,11 +111,13 @@
   "SqlServer": {
     "Initialize": "true",
     "AllowDatabaseCreation": "true",
-    "TransientFaultRetryPolicy": {
-      "InitialDelay": "00:00:00.100",
-      "RetryCount": 3,
-      "Factor": 2,
-      "FastFirst": true
+    "Retry": {
```

Comment:
Not sure why we do this change, could you share some insight

Reply:
Smitha made a change in the shared components library to refactor the retry config. Because it is unspecified today, it uses the default values, which I am now explicitly laying out in the config such that people can more easily change it in the future. See here: https://github.com/microsoft/healthcare-shared-components/pull/421/files#diff-0cd4095e857c7892723372de604da07479ffddd64a8a0c55d1411181bdc6ac6b